app/jobs/gias/sync_all_schools_job.rb

```diff
@@ -0,0 +1,15 @@
+module Gias
+  class SyncAllSchoolsJob < ApplicationJob
+    queue_as :default
+
+    def perform
+      tempfile = Gias::CsvDownloader.call
+      Rails.logger.info "GIAS CSV Downloaded!"
+
+      Rails.logger.info "Importing GIAS data"
+      Gias::Importer.call(tempfile.path)
```

Comment:
Would it be more consistent to call this `Gias::CsvImporter`?

```suggestion
      Gias::CsvImporter.call(tempfile.path)
```

Considering it imports the CSV file downloaded by the `Gias::CsvDownloader`.